pkg/apis/networking/ports_test.go

```diff
@@ -65,3 +65,26 @@ func TestServicePort(t *testing.T) {
 		})
 	}
 }
+
+func TestAppProtocol(t *testing.T) {
+	cases := []struct {
+		name   string
+		proto  ProtocolType
+		expect *string
+	}{{
+		name:   "pass h2c protocol to get Serving and Activator K8s services for HTTP/2 endpoints",
+		proto:  ProtocolH2C,
+		expect: &AppProtocolH2C,
+	}, {
+		name:   "other protocols result in nil",
+		proto:  ProtocolHTTP1,
+		expect: nil,
+	}}
+	for _, c := range cases {
+		t.Run(c.name, func(t *testing.T) {
+			if got, want := AppProtocol(c.proto), c.expect; !(got == want) {
```

Comment:
This is something trivial, but you can use `got != want`, instead of `!(got == want)`.